scenes/search.ts

```diff
@@ -12,7 +12,6 @@ let scene = new Scene<BotContext>('Search')
 
 scene.step(async (ctx) => {
   const message = HLanguage('searchRegion')
-  // FIXME: Refactor duplication on location.ts (15-29 lines)
```

Comment:
is it fixed?

Reply:
ne buni atadan keyinchali fix qilish uchun yozib qo’yganman. location.ts bilan duplicate qismi bor

Reply:
buni qaytarib qo'yaman bo'masam